packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -4029,7 +4029,7 @@ void main() {
 
     await gesture.up();
     await tester.pump();
-    expect(find.byType(CupertinoButton), findsNWidgets(3));
+    expect(find.byType(CupertinoButton), findsNWidgets(4));
```

Comment:
Nit: Seeing you change these numbers all over the place in these PRs makes me think we should have a better way to test this...  `expect(contextMenuButtonsShown(ContextMenuButtons.all));` or something like that.

That's probably beyond the scope of this PR, though.  Also it might not be worth it if we don't add many more buttons.  Just throwing out a thought.